core/genesis.go

```diff
@@ -57,6 +57,7 @@ func WriteGenesisBlock(chainDb ethdb.Database, reader io.Reader) (*types.Block,
 			Code    string
 			Storage map[string]string
 			Balance string
+			Nonce   string
```

Comment:
Does that mean it should remain a string inside the standard json spec? I mean converting one to the other is simple enough, just trying to get to a good representation.